internal/controller/grafanaorganization_controller.go

```diff
@@ -99,17 +101,79 @@ func (r GrafanaOrganizationReconciler) reconcileCreate(ctx context.Context, graf
 			return ctrl.Result{}, errors.WithStack(err)
 		}
 	}
-	_, err := grafanaAPI.Orgs.UpdateOrg(1, &grafanaAPIModels.UpdateOrgForm{
-		Name: "Shared Org.",
-	})
+
+	// if the CR doesn't have an orgID, create the organization in Grafana and update the status
+	if grafanaOrganization.Status.OrgID == "" {
+		return ctrl.Result{}, r.createOrganizationInGrafana(ctx, grafanaAPI, grafanaOrganization)
+	} else {
+		// Cast orgID to int64
+		orgID, err := strconv.ParseInt(grafanaOrganization.Status.OrgID, 10, 64)
```

Comment:
Why not make the field in the status an int64?

Reply:
As you can see here you can use int https://book.kubebuilder.io/cronjob-tutorial/api-design.html?highlight=int64 :D

Reply:
We definitely can make it an int :)

Reply:
Done